src/plugin.rs

```diff
@@ -72,9 +74,12 @@ impl Plugin for ShadPlayPlugin {
                 (
                     // utils::max_mon_res, // We're currently not using the maximum resolution of the primary monitor.
                     update_mouse_pos,
+                    toggle_border
+                        .run_if(in_state(AppState::TwoD))
```

Comment:
this is for `AppState::TwoD` so assuming it's the aforementioned is just a typo.